code/src/Form.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Header } from 'components/Header'
+import { FirstQuestion } from './components/FirstQuestion'
+import { SecondQuestion } from './components/SecondQuestion'
+import { ThirdQuestion } from './components/ThirdQuestion'
+import { SubmitButton } from './components/SubmitButton'
+import { Summary } from './components/Summary'
+import { useState } from 'react'
```

Comment:
we could add use state to the react import itseld i.e. import React, {useState} from 'react'